lib/rack/mock.rb

```diff
@@ -257,9 +257,13 @@ def identify_cookie_attributes(cookie_filling)
       cookie_bits.drop(1).each do |bit|
         if bit.include? '='
           cookie_attribute, attribute_value = bit.split('=', 2)
-          cookie_attributes.store(cookie_attribute.strip, attribute_value.strip)
           if cookie_attribute.include? 'max-age'
             cookie_attributes.store('expires', Time.now + attribute_value.strip.to_i)
```

Comment:
This appears to change behavior for `max-age`, where `it only stores `expires` instead of storing `expires` and `max-age`.  Is that intentional?

Reply:
Nice catch, it was not intentional. I will do as you suggested in https://github.com/rack/rack/pull/1753/files#r742316074 so it will once again be stored then.